src/ut/httpstack_test.cpp

```diff
@@ -100,7 +101,8 @@ class HttpStackTest : public testing::Test
   int get(const std::string& path,
           int& status,
           std::string& response,
-          std::list<std::string>* headers=NULL)
+          std::list<std::string>* headers=NULL,
+          bool add_body = false)
```

Comment:
Rather than this be a bool, can it be the body to add? It feels easier to extend that way